SnowplowTracker/SnowplowTracker/Emitter.cs

```diff
@@ -0,0 +1,473 @@
+/*
+ * Emitter.cs
+ * SnowplowTracker
+ * 
+ * Copyright (c) 2015 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ * 
+ * Authors: Joshua Beemster
+ * Copyright: Copyright (c) 2015 Snowplow Analytics Ltd
+ * License: Apache License Version 2.0
+ */
+
+using System;
+using System.Net;
+using System.Web;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Threading;
+using SnowplowTracker.Payloads;
+using SnowplowTracker.Enums;
+using SnowplowTracker.Requests;
+using SnowplowTracker.Storage;
+using UnityHTTP;
+
+namespace SnowplowTracker {
+	public class Emitter {
+
+		private int POST_WRAPPER_BYTES = 88;
+		private int POST_STM_BYTES = 22;
```

Comment:
I think there should be a comment saying where these numbers come from (or just directly define `POST_WRAPPER_BYTES = '{"schema":"iglu:com.snowplowanalytics.snowplow/payload_data/jsonschema/1-0-0","data":[]}'.length`).